test/integration/model/schema.test.js

```diff
@@ -0,0 +1,458 @@
+            expect(err).to.not.be.null;
+          });
+        });
+
+        it('should be able to insert data into the table in schema_two using create', function() {
+          var self = this;
+          var restaurantId;
+
+          return self.RestaurantTwo.create({
+            foo: 'two',
+            location_id: locationId
+          }).then(function() {
+              return self.RestaurantTwo.findOne({
+                where: {foo: 'two'}});
+            }).then(function(obj) {
+              expect(obj).to.not.be.null;
+              expect(obj.foo).to.equal('two');
+              restaurantId = obj.id;
+              return self.RestaurantTwo.findById(restaurantId);
+            }).then(function(obj) {
+              expect(obj).to.not.be.null;
+              expect(obj.foo).to.equal('two');
+            });
+        });
+
+
+        it('should fail to find schema_one object in schema_two', function() {
+          var self = this;
+
+          return self.RestaurantTwo.findOne({where: {foo: 'one'}}).then(function(RestaurantObj) {
+            expect(RestaurantObj).to.be.null;
+          });
+        });
+
+        it('should fail to find schema_two object in schema_one', function() {
+          var self = this;
+
+          return self.RestaurantOne.findOne({where: {foo: 'two'}}).then(function(RestaurantObj) {
+            expect(RestaurantObj).to.be.null;
+          });
+        });
+      });
+
+      describe('Add data via instance.save, retrieve via model.findAll', function() {
+        it('should be able to insert data into both schemas using instance.save and retrieve it via findAll', function() {
+          var self = this;
+
+          var restaurauntModel = self.RestaurantOne.build({bar: 'one.1'});
+
+          return restaurauntModel.save()
+            .then(function() {
+              restaurauntModel = self.RestaurantOne.build({bar: 'one.2'});
+              return restaurauntModel.save();
+            }).then(function() {
+              restaurauntModel = self.RestaurantTwo.build({bar: 'two.1'});
+              return restaurauntModel.save();
+            }).then(function() {
+              restaurauntModel = self.RestaurantTwo.build({bar: 'two.2'});
+              return restaurauntModel.save();
+            }).then(function() {
+              restaurauntModel = self.RestaurantTwo.build({bar: 'two.3'});
+              return restaurauntModel.save();
+            }).then(function() {
+              return self.RestaurantOne.findAll();
+            }).then(function(restaurantsOne) {
+              expect(restaurantsOne).to.not.be.null;
+              expect(restaurantsOne.length).to.equal(2);
+              restaurantsOne.forEach(function(restaurant) {
+                expect(restaurant.bar).to.contain('one');
+              });
+              return self.RestaurantOne.findAndCountAll();
+            }).then(function(restaurantsOne) {
+              expect(restaurantsOne).to.not.be.null;
```

Comment:
This block is saying the same thing as the previous, but with `findAndCount` right?